projects/kinesis/index.js

```diff
@@ -44,12 +44,12 @@ const gravUSDC = ADDRESSES.functionx.PURSE;
 const gravUSDT = ADDRESSES.functionx.USDT;
 
 module.exports = {
+  deadFrom: ['2023-03-01'],
```

Comment:
this should be a string not an array, but I think the root issue is, multicalls stopped working on evmos, maybe a new gas limit?

Reply:
I have just pushed a workaround